presto-native-execution/scripts/setup-centos.sh

```diff
@@ -33,8 +33,15 @@ export CC=/opt/rh/gcc-toolset-9/root/bin/gcc
 export CXX=/opt/rh/gcc-toolset-9/root/bin/g++
 
 CPU_TARGET="${CPU_TARGET:-avx}"
-SOURCE_FILE="$(dirname "${BASH_SOURCE}")/setup-helper-functions.sh"
-export COMPILER_FLAGS=$(source "$SOURCE_FILE" && echo -n $(get_cxx_flags $CPU_TARGET))
+SCRIPT_DIR=$(readlink -f "$(dirname "$0")")
```

Comment:
@Mionsz nit: I see we use `"${BASH_SOURCE[0]}"`  instead of `$0` in Velox and other places. Should we stick with that?

Reply:
It does not really matter - I will change it